README

```diff
@@ -1,7 +1,7 @@
 ReadMe - Ubuntu Clock App
 =========================
 
-Ubuntu Clock App is the official clock app for Ubuntu Touch. We follow an open
```

Comment:
Our product is still Ubuntu Touch, so this sentence is perfectly fine. The foundation is called UBports, but thats not what you are installing :)